README.md

```diff
@@ -17,138 +19,143 @@ Before getting started with this project, please get yourself familiar with the
+If successful, the output image file (adu-base-image-raspberrypi4-64.wic.gz) and example .swu update file (adu-update-image-raspberrypi4-64.swu) shold be located in `$build_output_dir/tmp/deploy/images/raspberrypi4-64` directory. If you built for version 0.0.0.1 you will need to copy the base file out and run the build again to produce a Sw Update update (file ending `.swu`) to be used for the update. You need to do this to make a usable base and update image. 
 
 ```sh
 .
-├── adu-base-image-raspberrypi3.wic.gz
-├── adu-update-image-raspberrypi3.swu
+├── adu-base-image-raspberrypi4-64.wic.gz
+├── adu-update-image-raspberrypi4-64.swu
 ```
 
 ## Build Pipelines Status
 
 | Board | Branch | Status |
 |---|---|---|
-| Raspberry Pi 3 | honister | [![Build Status](https://dev.azure.com/azure-device-update/adu-linux-client/_apis/build/status/azure.iot-hub-device-update-yocto?branchName=honister)](https://dev.azure.com/azure-device-update/adu-linux-client/_build/latest?definitionId=57&branchName=honister)|
+| Raspberry Pi 4 | kirkstone | [![Build Status](https://dev.azure.com/azure-device-update/adu-linux-client/_apis/build/status/azure.iot-hub-device-update-yocto?branchName=honister)](https://dev.azure.com/azure-device-update/adu-linux-client/_build/latest?definitionId=57&branchName=kirkstone)|
+
+
+## Using Your Own Board and Guidance for Production Images
+
+### Using Your Own Board
+
+If you've tried out Device Update on RaspberryPi 4 and decided you want to try and use it on other hardware you will need to port the `meta-raspberrypi-adu` layer to support your own board. You can find information on what changes may be required [here](https://github.com/Azure/meta-raspberrypi-adu/README.md). Keep in mind the `meta-raspberrypi-adu` layer is provided as is. It's a proof of concept. The repository contains information on how to port the existing proof of concept but you will likely need to add better u-boot scripts, include proper signing key information, and many other small things to get your board up to snuff. These are board dependent and are not under the purview of the Device Update team. If you have a question/comment please make a GitHub issue and we can take a look at it. 
+
+
+### Recommendations for Adapting for Production Images
+Like is said at the beginning of this document this repository is inteded to be a proof-of-concept. It is not intended to be a production ready drag and drop solution for building images to be used in the field. Within this repository We've made some recommendations for what might need to be changed but these recommendations should be taken as just that, recommendations. 
+
+
+## Question? Comment? Bug?
+
+Please create a GitHub issue and we'll get back to you as soon as we're able. Your feedback is integral to improving the agent, our software practices, and product direction. We're always happy to chat, but please RTFM (read the full manual) before posting. 
```

Comment:
_but please RTFM (read the full manual) before posting_

Hmmm.

Reply:
Fair.